src/BigQueryConnection.php

```diff
@@ -76,6 +76,9 @@ public function executeQuery(string $query): QueryResults
             if (str_contains($e->getMessage(), 'Job timed out after')) {
                 throw new UserException('Query exceeded the maximum execution time');
             }
+            if (str_contains($e->getMessage(), 'Job did not complete within the allowed number of retries.')) {
```

Comment:
Nejde k tomu dostat i původní error a ten třeba vrátit k tomu?
Např. tento job https://connection.us-central1.gcp.keboola.dev/admin/projects/18/queue/114285?eventId=20344395 :
<img width="1459" alt="image" src="https://github.com/keboola/google-bigquery-transformation/assets/903531/c22e0580-a5f1-464f-b029-4e1f7edb86c5">


Skončil

<img width="1202" alt="image" src="https://github.com/keboola/google-bigquery-transformation/assets/903531/de6da22c-01db-4c05-a82c-dfa48ebff07d">

Stejně se s toutou verzí např projeví https://keboola.atlassian.net/wiki/spaces/ENGG/pages/3100147728/BigQuery+-+Another+job+is+currently+running+in+the+session

Reply:
Myslim ze tady na to handlovani "Job did not complete within the allowed number of retries" se bude chtit jeste podivat, zda se ze obcas skonci drive nez by melo. Chyba vyse se ve verzi bez tohoto neobjevuje.

Naopak ten fix `invalid_grant` vypada ok.

Reply:
podle me to bude i shazovat delsi joby kvuli https://github.com/googleapis/google-cloud-php-bigquery/blob/a918523062c78d8fd29422e873f72c59ee97bbe0/src/JobWaitTrait.php#L54 Haze to pri kazdem nedokoncenem jobu